release/release_tests.yaml

```diff
@@ -1506,12 +1416,11 @@
 
   frequency: nightly
   team: ml
-  env: staging
+  env: staging_v1
```

Comment:
so this is still on v1?

Reply:
yes, as outlined in description - we'll track those as issues and assign test owners to fix